pkg/codegen/internal/test/testdata/aws-native-pseudo-params-pp/aws-native-pseudo-params.pp

```diff
@@ -0,0 +1,57 @@
+awsPartition = invoke("aws-native::getPartition").partition
+
+config trailName string {
+}
+
+config bucketName string {
+}
+
+isOrganizationsSupported = awsPartition == "aws"
+
+resource trail "aws-native:cloudtrail:Trail" {
+	s3BucketName = bucketName
+	s3KeyPrefix = "Uluru"
+	isLogging = true
+	trailName = trailName
+	enableLogFileValidation = true
+	includeGlobalServiceEvents = true
+	isMultiRegionTrail = true
+	cloudWatchLogsLogGroupArn = invoke("aws-native::importValue", {
+		name = "TrailLogGroupTestArn"
+	}).value
+	cloudWatchLogsRoleArn = invoke("aws-native::importValue", {
+		name = "TrailLogGroupRoleTestArn"
+	}).value
+	kmsKeyId = invoke("aws-native::importValue", {
```

Comment:
Program generation breaks for me unless I specify "kMSKeyId" here...but only node code generation emits a legible error message. interesting..